internal/helpers/generators.go

```diff
@@ -14,32 +15,34 @@ func GenerateDatabaseFile(folderPath string, provider Provider) error {
 package initializers
 
 import (
-	"fmt"
-	{{range .Imports}}
-	{{.}}
-	{{- end}}
+    "fmt"
+    {{range .Imports}}
+    {{.}}
+    {{- end}}
 )
 
 var DB {{.DBVariable}}
 
 func ConnectDB(){
-	{{.ConnectionCode}}
+    {{.ConnectionCode}}
 }
 `)
 	if err != nil {
-		return 
+		log.Printf("error parsing database template: %v", err)
+		return fmt.Errorf("error parsing database template: %v", err)
```

Comment:
no need of both log.printf and fmt.errorf as it will log the error two times, keep one

Reply:
Okk